.changeset/cold-bears-sneeze.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/markdown-remark': patch
```

Comment:
the fact that this could break some existing URL hashes means that we'll need to:

- think this through a bit more than normal: we want to make this change once and only once, if we can help it.
- time it with other breaking changes, if we have any to go out together
- message it as a potentially breaking change in the changelog
-

Reply:
Hmm. You are right Fred. But then should it be a `minor` or `major` change? Will there be a `v0.27`?